content/_partials/download.html.haml

```diff
@@ -52,7 +52,7 @@
               %img{:src => "/images/os/opensuse.png"}
                 %a.release-block-soft{:href => "http://pkg.jenkins-ci.org/opensuse/"} openSUSE
             %div
-              %img{:src => "/images/os/freebsd.png"}
+              %img{:src => "/sites/default/files/images/freebsd.png"}
```

Comment:
It looks like that file only exists due to the `copyLegacyAssets` task? Could we just replace the existing `/images/os/freebsd.png` image with the horns one? Or does it not matter as the new design will look different here?